src/app/shared/config/config.service.ts

```diff
@@ -40,22 +45,41 @@ export const lowClone = <T>(obj: T): T => {
   return clone;
 };
 
+const defaultOidcConfig = {
```

Comment:
Suggest renaming this const to `placeholderOidcConfig` for consistency of it's usage

Reply:
Was reverted. Resolved.